package.json

```diff
@@ -15,18 +15,40 @@
     "email": "gsaper@gmail.com"
   },
   "keywords": [
-    "gruntplugin", "image", "image-size"
+    "gruntplugin",
+    "image",
+    "image-size"
   ],
   "engines": {
     "node": ">= 0.8.0"
   },
-  "licenses": [
-    {
-      "type": "MIT"
-    }
+  "main": "tasks/grayMatter.js",
+  "files": [
+    "tasks"
   ],
+  "licenses": "MIT",
   "dependencies": {
     "chalk": "^0.5.0",
-    "image-size": ""
+    "image-size": "^0.6.2"
+  },
+  "devDependencies": {
+    "eslint": "^4.13.1",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-prettier": "^2.4.0",
+    "grunt": "^1.0.1",
+    "grunt-contrib-clean": "^1.1.0",
+    "jest": "21.2.1",
+    "jit-grunt": "^0.10.0",
+    "prettier": "^1.9.2"
+  },
+  "scripts": {
+    "lint": "eslint \"./**/*.js\"",
+    "format": "eslint \"./**/*.js\" --fix",
```

Comment:
I've mentioned it in PR description. I didn't do it because wasn't sure which path would you prefer.

My imho — it is way more practical to lint Gruntfile and tests too. So I'm adding node and es6 to ESlint env than.